applications/notify/src/notify_fax_util.erl

```diff
@@ -50,8 +50,12 @@ get_attachment(UseDb, Category, Props) ->
     {'ok', AttachmentBin, ContentType} = raw_attachment_binary(UseDb, FaxId),
 
     case kapps_config:get_binary(Category, <<"attachment_format">>, <<"pdf">>) of
-        <<"pdf">> -> convert_to_pdf(AttachmentBin, Props, ContentType);
-        _Else -> convert_to_tiff(AttachmentBin, Props, ContentType)
+        <<"pdf">> ->
+            case kz_convert:fax(ContentType, <<"application/pdf">>, AttachmentBin, [{<<"output_type">>, 'binary'}]) of
```

Comment:
why converting again? see what `attachment_format` is configured to and use correct `kzd_fax` fetch function.